src/vs/workbench/parts/extensions/electron-browser/extensionsActions.ts

```diff
@@ -1156,18 +1157,60 @@ export class DisableAction extends Action {
 export class CheckForUpdatesAction extends Action {
 
 	static readonly ID = 'workbench.extensions.action.checkForUpdates';
-	static LABEL = localize('checkForUpdates', "Check for Updates");
+	static LABEL = localize('checkForUpdates', "Check for Extension Updates");
 
 	constructor(
 		id = UpdateAllAction.ID,
 		label = UpdateAllAction.LABEL,
-		@IExtensionsWorkbenchService private extensionsWorkbenchService: IExtensionsWorkbenchService
+		@IExtensionsWorkbenchService private extensionsWorkbenchService: IExtensionsWorkbenchService,
+		@IViewletService private viewletService: IViewletService,
+		@INotificationService private notificationService: INotificationService
 	) {
 		super(id, label, '', true);
 	}
 
+	private onUpdateNotAvailable(): void {
```

Comment:
For brevity purposes, I would suggest to combine the `onUpdateNotAvailable` and `onUpdateAvailable` functions to a single function `notifyUpdateAvailablility` and make it take the outdated count as a parameter.

Reply:
Also, I just realized that it would be cool if we present an option to `Update all outdated extensions` in this notification. Then we can skip the change of focus to the extension viewlet.

Reply:
>Also, I just realized that it would be cool if we present an option to Update all extensions in this notification. Then we can skip the change of focus to the extension viewlet.

I also thought of the same prompt while coding and redirecting to Update all extensions. Would be a nice to have option 👍